### PR TITLE
15740 - AUTH Web - handle magic link affiliation for users with multiple BCROS accounts

### PR DESCRIPTION
**Pull request by Jxio**
*Issue #:*
https://github.com/bcgov/entity/issues/15740

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/components/auth/manage-business/EntityManagement.vue [hunk 1]
```diff
@@ -416,6 +443,48 @@ export default class EntityManagement extends Mixins(AccountMixin, AccountChange
...
+        // 1. link expired
+        if (invitation.data.status === AffiliationInvitationStatus.Expired) {
+          this.showLinkExpiredModal(identifier)
+          return
+        }
+
+        // 2. business already added
+        const isAdded = await this.isAffiliated(identifier)
+        const business = await BusinessService.searchBusiness(identifier)
+        if (isAdded && business && business.data && business.status === 200) {
```
Comment (seeker25) [inline auth-web/src/components/auth/manage-business/EntityManagement.vue:468]: I'd almost wrap these in the service itself. Similar to what we do in `business-edit-ui`

Comment (seeker25) [reply auth-web/src/components/auth/manage-business/EntityManagement.vue:468]: https://github.com/bcgov/business-edit-ui/blob/ef4a03ba0fc64f7b0297fd3aa46d9aaa39eadd18/src/services/auth-services.ts#L66 

Can still use async await and not `.then`

Comment (seeker25) [inline auth-web/src/components/auth/manage-business/EntityManagement.vue:468]: What happens if it's not 200?

**Review by seeker25 — APPROVED** [review]
Minor suggestions

File: auth-web/src/components/auth/manage-business/EntityManagement.vue (hunk 1 above)
Comment (Jxio) [reply auth-web/src/components/auth/manage-business/EntityManagement.vue]: I removed business, the business name should in the email link

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2434-lw397n5h.web.app

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2434-lw397n5h.web.app

**Pull request merged**